<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CovariateItemWidget</class>
 <widget class="QWidget" name="CovariateItemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>506</width>
        <height>528</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="generalBox">
         <property name="title">
          <string>General</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="idLabel">
            <property name="text">
             <string>ID</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="idLine"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="nameLabel">
            <property name="text">
             <string>Name</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="TranslatableLineEdit" name="nameLine"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="descriptionLabel">
            <property name="text">
             <string>Description</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="TranslatablePlainTextEdit" name="descriptionText"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="definitionBox">
         <property name="title">
          <string>Definition</string>
         </property>
         <layout class="QFormLayout" name="definitionLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="unitLabel">
            <property name="text">
             <string>Unit</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="UnitComboBox" name="unitBox"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="defaultLabel">
            <property name="text">
             <string>Default</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="typeLabel">
            <property name="text">
             <string>Type</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="typeBox"/>
          </item>
          <item row="3" column="1">
           <widget class="QWidget" name="defaultWidget" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TranslatablePlainTextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header>translatableplaintextedit.h</header>
  </customwidget>
  <customwidget>
   <class>UnitComboBox</class>
   <extends>QComboBox</extends>
   <header>unitcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>TranslatableLineEdit</class>
   <extends>QLineEdit</extends>
   <header>translatablelineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
